sdk/agricultureplatform/arm-agricultureplatform/src/static-helpers/urlTemplate.ts

```diff
@@ -170,7 +170,7 @@ export function expandUrlTemplate(
     }
     let op;
     if (["+", "#", ".", "/", ";", "?", "&"].includes(expr[0])) {
-      (op = expr[0]), (expr = expr.slice(1));
+      ((op = expr[0]), (expr = expr.slice(1)));
```

Comment:
Thanks for this improvement  and i plan to have a fix for this in codegen side in this issue: https://github.com/Azure/autorest.typescript/issues/3360.